corehq/apps/cloudcare/static/cloudcare/js/form_entry/entries.js

```diff
@@ -1099,6 +1100,19 @@ hqDefine("cloudcare/js/form_entry/entries", function () {
                     entry = new ChoiceLabelEntry(question, {
                         hideLabel: hideLabel,
                     });
+                    if (!hideLabel) {
+                        kissmetrics.track.event("Accessibility Tracking - Tabular Question Seen");
```

Comment:
I might have implemented `required` as a property on the "seen" event instead of a separate event. Thoughts on that? Ultimately whatever you and product decide is fine, I'm not sure what'll be easiest for them to build reports off of.

Reply:
I think that makes a lot of sense. No need for a separate event.